src/data/markdown/translated-guides/en/02 Using k6/08 Tags and Groups.md

```diff
@@ -26,7 +26,7 @@ k6 provides two types of tags:
 - *System tags* are tags that k6 automatically assigns.
 - *User-defined* tags are tags that you add when you write your script.
 
-### System tags
+## System tags
```

Comment:
What was the reasoning behind bringing the system tags section as the same level as tags? 😸

Reply:
The reason was for creating the right bar with all of this section with a specific link. AFAIK, only the h2 headers are added to the list.

![Screenshot from 2022-05-02 12-10-41](https://user-images.githubusercontent.com/2103732/166218657-3a5e143c-539e-48c0-bafb-65af9acc35d1.png)

Reply:
Makes sense. Thanks for the clarification ❤️